pagarme/former-kit
# table: add loader table row component

## Context
This PR add `TableLoaderRow` component to handle with loading states in table.

## Linked Issues
- [ ] Resolves #237 


### Preview:
![table-loader](https://user-images.githubusercontent.com/6943919/53980735-d4057680-40ef-11e9-8c65-637eb9b71b88.gif)

### How to test:

1. Clone this repository in this branch `git clone -b add/table-loader-row git@github.com:pagarme/former-kit` or checkout to this branch `git checkout add/table-loader-row`
2. make sure you have installed all dependencies: `yarn`
3. run the the storybook: `yarn storybook`
